tests/console/ncurses.pm

```diff
@@ -24,6 +24,8 @@ use utils qw(clear_console zypper_call);
 sub run {
     select_console 'root-console';
     zypper_call 'in dialog';
+    #poo 88033, we need to clean up console before issue command.
+    clear_console;
```

Comment:
I think should before zypper_call 'in dialog', as 'in dialog' always has problem.

Reply:
ok, will try add another one then reply you.

Reply:
in my most recently 4 runs add one before dialog is enough.